var/spack/repos/builtin/packages/netcdf-cxx4/package.py

```diff
@@ -0,0 +1,16 @@
+from spack import *
+
+class NetcdfCxx4(Package):
+    """C++ interface for NetCDF4"""
+    homepage = "http://www.unidata.ucar.edu/downloads/netcdf/netcdf-cxx/index.jsp"
```

Comment:
I would just use the same homepage as NetCDF for NetCDF-CXX4 and NetCDF-Fotran:
http://www.unidata.ucar.edu/software/netcdf/